integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -301,7 +302,9 @@ def test_mod_pmod_by_zero(data_gen, overflow_exp):
 def test_cast_neg_to_decimal_err():
     # -12 cannot be represented as decimal(7,7)
     data_gen = _decimal_gen_7_7
-    exception_content = "Decimal(compact,-120000000,20,0}) cannot be represented as Decimal(7, 7)"
+    dec_value = "Decimal(compact,-120000000,20,0})" if is_before_spark_330() \
+        or is_databricks_runtime() else "Decimal(compact, -120000000, 20, 0)"
```

Comment:
Will this check fail again if databricks has a version of spark 330?

Reply:
I believe yes